### PR TITLE
Fix docs root redirect

### PR DESCRIPTION
**Pull request by cherewaty**
## Motivation
Visiting https://frontside.com/interactors redirects to https://frontside.com/interactors/interactors.

## Approach
Move the old `interactors` page to the index.

### CODE REVIEW

**Comment by cowboyd** [comment]
This looks good, but I'll defer to @jorgelainfiesta and @minkimcello for final approval. Things I've noticed:

1. We still have a CTA to sign up for the BigTest email list. Do we want to have that?
2. In the "why interactors" section, we should have a link to existing interactors piece.
3. Should the "Beta" tag be on Interactors, not Docs? It feels like the docs are beta
4. We have great typedoc for interactors. We should begin to render it.

**Comment by cherewaty** [comment]
Removed BigTest newsletter form. We can tackle other design and content pieces next week, after merge.

**Review by minkimcello — APPROVED** [review]

**Pull request merged**
